### PR TITLE
Refactor

### PR DESCRIPTION
**Pull request by tjvignos**
## Overview

Refactored code base to remove unnecessary code, add crud endpoints for all models, and remove unnecessary data returned in json.

## Changes Made

Aayush: Revamped cron/docker files, changed views/urls/models.
Daniel: Added crud endpoints and corresponding url files.
Thomas: Removed redundant ids from serializers to reduce load times.

## Test Coverage

Tested locally.

### CODE REVIEW

**Review by mateow99 — APPROVED** [review]
LGTM

File: src/cron/update_db.txt
Comment (kidzegeye) [inline src/cron/update_db.txt:1]: Need to look into the hours on this cronjob, the timezone is UTC

File: src/eatery/models.py [hunk 1]
```diff
@@ -10,10 +10,9 @@ class CampusArea(models.TextChoices):
         COLLEGETOWN = "Collegetown"
         NONE = ""
 
-    id = models.IntegerField(primary_key=True)
     name = models.CharField(max_length=40)
     menu_summary = models.TextField(blank=True, null=True, default="")
-    image_url = models.URLField(blank=True)
+    image_url = models.URLField(blank=True, default="https://images-prod.healthline.com/hlcmsresource/images/AN_images/health-benefits-of-apples-1296x728-feature.jpg")
```
Comment (kidzegeye) [inline src/eatery/models.py]: The default img url should be in a constant variable

**Review by kidzegeye — APPROVED** [review]
Left a few small comments but lgtm

**Pull request merged**